### DISCUSSION TITLE
HMS-800-2T startet nicht

### DISCUSSION THREAD

**Discussion by Fokker-1** [tbnobody/OpenDTU#2017] [Q&A]
Hallo,

ich habe mir als weiteren WR (ein HM-800 läuft bereits seit einem Jahr problemlos) einen HM**S**-800-2T (den ohne "W") zugelegt und getestet.

Folgende Tests habe ich vorab gemacht, da nach Installation der WR nur noch schwer zugänglich ist.

**1. WR Einschalten**
Netzspannung ist abgeschaltet.
Dann lege ich an einen der Ports eine Spannung von 17 V aus einem Labornetzgerät an. Gemäß Datenblatt soll sich der WR ab 16 V einschalten. 

Die LED blinkt nach kurzer Zeit 5x grün und schnell mit mit 3 Impulsen / Sekunde und wechselt dann zu einem schnellen, roten Blinken mit 2 Impulsen / Sekunde. Das funktioniert so auch immer zuverlässig, schon mal prima.
Dann erhöhe ich die Spannung auf 24 V, denn gemäß Datenblatt soll der WR ab 22 V mit der Stromeinspeisung ins 230 V Netz starten.

**2 . Stromeinspeisung ins 230 V Netz**
Dann schalte ich die Netzspannung an den WR.
Nach kurzer Zeit wechselt die rote LED zum Blikzyklus 1x / Sekunde
Es sollte gemäß Datenblatt nach etwa 2 Minuten die Einspeisung beginnen und die LED mit grünem Impulse mit 1 Impuls / Sekunde blinken.
Aber genau das passiert i.d.R. nicht, die LED bleibt im Blinkzyklus 1x je Sekunde rot. Von etwa 10 Tests dieser Art startet der WR mit der Einspeisung in höchstens nur 2 Fällen - wenn überhaupt.

Ich habe nun im Netz Berichte gefunden, die genau dieses Problem ebenfalls beschreiben. Das war bei dem UMS-800**W**-2T. Soweit ich das verstanden habe, wurde über die Original Hoymiles DTU ein SW-Update aufgespielt und das Netzprofil angepasst. Aber das Teil habe ich nicht, werde es mir auch nicht kaufen, ist sehr teuer - und ich will nicht in die Cloud.

Aktuell habe ich keine Idee, was ich noch testen könnte.
Gibt es hier Erfahrungen zum Problem und evtl. hilfreiche Infos?

Schon mal vielen Dank im Voraus.

Grüße
Werner

**Comment by ghost** [comment]
Hi,
Wenn das Blinken der LED eindeutig ist und du alles entsprechend der Anleitung beim Inbetriebnehmen richtig gemacht hast, kannst du dich wegen dem defekten WR doch auch an deinen Verkäufer wenden.
Keiner kann von dir verlangen, dass du dir eine originale DTU besorgst du die fehlerhafte Firmware selbst korrigiert.
Du  hast ein funktionstüchtige Gerät erwerben wollen, jetzt  ist der Händler mit nachbessern in der Pflicht.

**Comment by Fokker-1** [comment]
Hallo schneer,

danke für die Hinweise.

Die Rückgabebedingungen sind mir soweit klar, 30 Tage nach Kauf kostenfrei zurück ohne Begründung.
Ich hatte gehofft, dass es noch irgendeinen technischen Hinweis zur Problemlösung gibt, bevor ich die nervige Rückgabe und Neubestellung starte.

Klar auch, Original-DTU ist für mich absolut kein Thema.

Gruß
Werner
